docs/java-api/docs/update.asciidoc

```diff
@@ -87,7 +87,7 @@ IndexRequest indexRequest = new IndexRequest("index", "type", "1")
 UpdateRequest updateRequest = new UpdateRequest("index", "type", "1")
         .doc(jsonBuilder()
             .startObject()
-                .field("gender", "male")
+                .field("name", "Joe Dalton")
```

Comment:
actually I think this was ok as if you look below it says
```
<1> This field is added by the update request
```
however a bit further up, we should remove the  field on `indexRequest`
```
                .field("gender", "male")
```
wdyt? Can you please update this?

Reply:
since this statement explains the `update` operation, I think it's better to update some field.
mark the `name` field as
```
 <1> This field is updated by the update request
```
should make it clear now.